runtime/near-vm-logic/Cargo.toml

```diff
@@ -43,29 +43,29 @@ costs_counting = []
 
 [[test]]
 name = "test_storage_read_write"
-path = "tests/test_storage_read_write.rs"
+path = "src/tests/test_storage_read_write.rs"
 
 [[test]]
 name = "test_context"
-path = "tests/test_context.rs"
+path = "src/tests/test_context.rs"
 
 [[test]]
 name = "test_miscs"
-path = "tests/test_miscs.rs"
+path = "src/tests/test_miscs.rs"
 required-features = ["costs_counting"]
 
 [[test]]
 name = "test_registers"
-path = "tests/test_registers.rs"
+path = "src/tests/test_registers.rs"
 
 [[test]]
 name = "test_storage_usage"
-path = "tests/test_storage_usage.rs"
+path = "src/tests/test_storage_usage.rs"
 
 [[test]]
 name = "test_promises"
-path = "tests/test_promises.rs"
+path = "src/tests/test_promises.rs"
 
 [[test]]
 name = "test_iterators"
-path = "tests/test_iterators.rs"
+path = "src/tests/test_iterators.rs"
```

Comment:
Shouldn’t those be deleted now?

Reply:
Thanks, good observation. I deleted those lines.